mass_mailing_partner/models/res_partner.py

```diff
@@ -14,7 +14,6 @@ class ResPartner(models.Model):
     mass_mailing_contact_ids = fields.One2many(
         string="Mailing contacts",
         oldname="mass_mailing_contacts",
```

Comment:
This should go away

Reply:
> This should go away

@chienandalu why? We should put reasons please :busts_in_silhouette: :1234: :smile:

Reply:
@rafaelbn oldaname is used by Odoo to rename fields automatic DB tables when upgrade modules or migration happens then if the migration occurs in order, it should be like from 9 to 10 then from 10 to 11, etc.. 
This means here makes no sense because this oldname could only exist before version 10 (is the origin of this module migration)